.github/workflows/build-and-deploy.yml

```
@@ -61,13 +61,16 @@ jobs:
           asset_content_type: application/zip
 
   create-release:
-    needs: build # This job depends on the build job
+    needs: [ build, upload-release-asset ] # This job depends on the build job
```

yamllint

[error] too many spaces inside brackets

(brackets)